misc/lldb_rb/commands/heap_page_command.py

```diff
@@ -0,0 +1,38 @@
+import lldb
+
+from lldb_rb.constants import *
+from lldb_rb.rb_base_command import RbBaseCommand
+
+class HeapPageCommand(RbBaseCommand):
+    program = "heap_page"
+    help_string = "prints out 'struct heap_page' for a VALUE pointer in the page"
+
+    def __init__(self, debugger, _internal_dict):
+        self.internal_dict = _internal_dict
+
+    def __call__(self, debugger, command, exe_ctx, result):
+        super().__call__(debugger, command, exe_ctx, result)
+        target, _process, _thread, frame = self.build_environment(debugger)
+
+        self.tHeapPageBody = target.FindFirstType("struct heap_page_body")
+        self.tHeapPagePtr = target.FindFirstType("struct heap_page").GetPointerType()
```

Comment:
We should standardize on snake case.

Reply:
Fair. This was copied from an existing command as is, but I'll change as part of this branch.